_includes/getstarted.html

```diff
@@ -16,25 +16,29 @@
                 <pre class="console"><span>&gt;</span> flyway migrate -url=... -user=... -password=...</pre>
                 <a href="/documentation/getstarted/firststeps/commandline" class="btn btn-primary">Get Started with the
                     Command-line Tool <i class="fa fa-arrow-right"></i></a>
+                <a class="btn btn-info" href="/documentation/getstarted/how">How Flyway works <i class="fa fa-arrow-right"></i></a>
```

Comment:
It feels strange having this button appear 5 times on one page now. Shouldn't the button in the middle of the page be enough?

Reply:
It doesnt appear 5 times, the 4 locations are next to the 'get started with' buttons:

![image](https://user-images.githubusercontent.com/1607501/95571197-f70cb080-0a1f-11eb-9b32-1c276e701d49.png)